### PR TITLE
Move nav to src

### PR DESCRIPTION
**Pull request by zeroadam**
Move include/nav to src Nav class

Related to #3878

### CODE REVIEW

**Comment by zeroadam** [comment]
I started this before @MrPetovan 's huge PR was merged. I rebased, so hopefully the array syntax in Nav is still correct!

**Comment by MrPetovan** [comment]
If it's not, I'll be the first to notice it!

**Comment by zeroadam** [comment]
👍

File: src/Content/Nav.php [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+	
+		if (count($a->apps) > 0) {
+			$nav['apps'] = ['apps', t('Apps'), '', t('Addon applications, utilities, games')];
+		}
+	
+		if (local_user() || !Config::get('system', 'local_search')) {
+			$nav['search'] = ['search', t('Search'), '', t('Search site content')];
+	
+			$nav['searchoption'] = [
+							t('Full Text'),
```
Comment (MrPetovan) [inline src/Content/Nav.php]: Standards: Wrong indentation.

File: src/Content/Nav.php [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+		$nav_info = self::getInfo($a);
+	
+		/*
+		 * Build the page
+		 */
+	
+		$tpl = get_markup_template('nav.tpl');
+	
+		$a->page['nav'] .= replace_macros($tpl, [
+				'$baseurl' => System::baseUrl(),
```
Comment (MrPetovan) [inline src/Content/Nav.php]: Standards: Wrong indentation.

File: src/Content/Nav.php [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+		$nav['about'] = ['friendica', t('Information'), '', t('Information about this friendica instance')];
+	
+		// The following nav links are only show to logged in users
+		if (local_user()) {
+			$nav['network'] = ['network', t('Network'), '', t('Conversations from your friends')];
+			$nav['net_reset'] = ['network/0?f=&order=comment&nets=all', t('Network Reset'), '', t('Load Network page with no filters')];
+	
+			$nav['home'] = ['profile/' . $a->user['nickname'], t('Home'), '', t('Your posts and conversations')];
+	
+			if (in_array($_SESSION['page_flags'], array(PAGE_NORMAL, PAGE_SOAPBOX, PAGE_FREELOVE, PAGE_PRVGROUP))) {
```
Comment (MrPetovan) [inline src/Content/Nav.php]: `[...]`
Standards: Wrong indentation.

File: src/Content/Nav.php [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+				// only show friend requests for normal pages. Other page types have automatic friendship.
+				if (in_array($_SESSION['page_flags'], array(PAGE_NORMAL, PAGE_SOAPBOX, PAGE_PRVGROUP))) {
```
Comment (MrPetovan) [inline src/Content/Nav.php]: `[...]`

File: src/Content/Nav.php [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+					$nav['introductions'] = array('notifications/intros', t('Introductions'), '', t('Friend Requests'));
```
Comment (MrPetovan) [inline src/Content/Nav.php]: `[...]`

File: src/Content/Nav.php [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+				}
+				if (in_array($_SESSION['page_flags'], array(PAGE_NORMAL, PAGE_SOAPBOX, PAGE_FREELOVE))) {
```
Comment (MrPetovan) [inline src/Content/Nav.php]: `[...]`

**Pull request merged**